examples/azure_file/azure.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: azure
+spec:
+ containers:
+  - image: kubernetes/pause
+    name: azure
+    volumeMounts:
+      - name: azure
+        mountPath: /mnt/azure
+ volumes:
+      - name: azure
+        azureFile:
+          accountName: account_name
+          accountKey: account_key
```

Comment:
Hmm.. other implementations `gce_pd` and `aws_ebs` for example have kept account information outside the volume definition. Is it possible to do that with Azure ? Perhaps there is a way to authorize the instance through the Azure. I think that would be a better long term strategy especially after the implementation of the Attach/Detach controller.